src/formatter/helpers/issue_helpers.js

```diff
@@ -171,3 +173,15 @@ export function formatIssue({
   })
   return `${text}\n`
 }
+
+function getRetryWarningText(testCase, flakyColorFn) {
+  const result = testCase.result
+  if (!testCase.result) {
+    return ''
+  }
+  const attemptNumber = testCase.attemptNumber
+  if (attemptNumber > 1 || result.status === Status.RETRY) {
+    return flakyColorFn(`(attempt #${attemptNumber}) `)
+  }
+  return ''
+}
```

Comment:
Can we please add a test for this update?

Reply:
Sure

Reply:
Yep